dialog/src/main/java/roboy/talk/Verbalizer.java

```diff
@@ -120,6 +120,10 @@ private SpeechAction anecdote(Interpretation interpretation){
 			new RandomList<>("Oh no, I didn't get what you said. ", "I didn't understand you correctly. ", "Sorry? What did you say? "
 			);
 
+	public static final RandomList<String> rosDisconnect =

```

Comment:
let's not tell the user that our ROS isnt working :) my suggestion is to create a least of generic phrases for exceptions like "oopsie, i need to run some internal checkups, my code needs attention" or "oh-oh, my central processing unit needs some cooling right now"

in addition, make sure in the logger you notify a developer about that specific issue, ROS in this case